src/distilabel/utils/dataset.py

```diff
@@ -51,3 +60,215 @@ def load_task_from_disk(path: Path) -> "Task":
     with open(task_path, "rb") as f:
         task = pickle.loads(f.read())
     return task
+
+
+def _binarize_dataset(
+    dataset: "CustomDataset",
+    seed: int = None,
+    strategy: BinarizationStrategies = "random",
+    keep_ties: bool = True,
```

Comment:
why did we set `keep_ties` to `True` as default? Intuitively I would say removing them will lead to higher quality data.

Reply:
Wasn't sure what value to set as default, and thought that allowed to keep all the data initially, but yes, I think it makes more sense to remove those by default